### ISSUE TITLE
 Incorrect field "self.bank_account_id" name. It should be "self.partner_bank_iban".

### ISSUE THREAD

**Issue by ieski** [OCA/account-reconcile#468]
https://github.com/OCA/account-reconcile/blob/c1ecba90e74adff44146f81aa0f5733ae30c149e/account_reconciliation_widget/models/account_bank_statement.py#L181

**Comment by pedrobaeza** [comment]
Odoo version? Steps to reproduce the problem?

**Comment by ieski** [comment]
Version: v.15
Line: 181

**It will change with this**
`if self.account_number and self.partner_id and not self.partner_bank_id :`

**Comment by pedrobaeza** [comment]
I see `bank_partner_id` and `partner_bank_id` on the model. Are you sure the one you point is the correct?

**Comment by ieski** [comment]
Ok. but no "bank_account_id"

```python
        if self.account_number and self.partner_id and not self.bank_account_id:
            # Search bank account without partner to handle the case the res.partner.bank
            # already exists but is set on a different partner.
            self.partner_bank_id = self._find_or_create_bank_account()
```

**Comment by pedrobaeza** [comment]
Then which one is the correct?

**Comment by ieski** [comment]
This correct:

```
        if self.account_number and self.partner_id and not self.partner_bank_id:
            # Search bank account without partner to handle the case the res.partner.bank
            # already exists but is set on a different partner.
            self.partner_bank_id = self._find_or_create_bank_account()
```

**Comment by pedrobaeza** [comment]
Fixed in b44d9d4
